<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>windowYes</class>
 <widget class="QDialog" name="windowYes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>247</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>American Typewriter</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>What do you need help with?</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>631</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="Button_Networking">
      <property name="text">
       <string>Networking</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_Scholarships">
      <property name="text">
       <string>Scholarships</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>591</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="Button_Motivation">
      <property name="text">
       <string>Motivation</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="info1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>311</width>
     <height>111</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Klik what you need help with to see more&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next1Button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>320</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CodeGirls">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>0</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>CodeGirls</string>
   </property>
  </widget>
  <widget class="QComboBox" name="NetworkingBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>311</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Instagram - Women who code</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Instagram - Girls who code </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Instagram - 10 Women In Tech To Follow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LinkedIn - Women who code</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LinkedIn - Women in IT and Tech</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LinkedIn - International Women in Tech</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Facebook - Women In Tech</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Facebook - Black Women In Tech</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Twitter - WomenTech Network</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Twitter - Women Who Code</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Twitter - Girls in Tech</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Twitter - AnitaB.org</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18 organizations advancing women in tech</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>479</x>
     <y>500</y>
     <width>247</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>American Typewriter</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>What do you need help with?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next1Button_2">
   <property name="geometry">
    <rect>
     <x>629</x>
     <y>740</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Motivation_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>420</y>
     <width>589</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Motivation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CodeGirls_2">
   <property name="geometry">
    <rect>
     <x>819</x>
     <y>420</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>CodeGirls</string>
   </property>
  </widget>
  <widget class="QComboBox" name="NetworkingBox_2">
   <property name="geometry">
    <rect>
     <x>449</x>
     <y>710</y>
     <width>311</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Instagram - Women who code</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Instagram - Girls who code </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Instagram - 10 Women In Tech To Follow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LinkedIn - Women who code</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LinkedIn - Women in IT and Tech</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LinkedIn - International Women in Tech</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Facebook - Women In Tech</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Facebook - Black Women In Tech</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Twitter - WomenTech Network</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Twitter - Women Who Code</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Twitter - Girls in Tech</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Twitter - AnitaB.org</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18 organizations advancing women in tech</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>289</x>
     <y>540</y>
     <width>631</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="Button_Networking_2">
      <property name="text">
       <string>Networking</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_Scholarships_2">
      <property name="text">
       <string>Scholarships</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="info1_2">
   <property name="geometry">
    <rect>
     <x>449</x>
     <y>660</y>
     <width>311</width>
     <height>111</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Klik what you need help with to see more&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
